task7/index.js

```diff
@@ -0,0 +1,45 @@
+const obj = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+const div = document.getElementById("js-parent");
+
+const loading = () => {
+    const loadingPlace = document.createElement("div");
+    const gif = document.createElement('img');
+    loadingPlace.id = "loadingPlace";
+    gif.src = "loading-circle.gif";
+    div.appendChild(loadingPlace).appendChild(gif);
+};
+    
+const getData = new Promise((resolve) => {
+    loading();
+    setTimeout(() => {
+        resolve(obj);
+    },2000);
+});
+
+getData.then(() => {
+    const loadingPlace = document.getElementById("loadingPlace");
+    loadingPlace.remove();
+});
```

Comment:
nits
This is a really small detail that doesn't need to be modified, but please keep in mind that you can cut this out as a function as well.
If you do so, the code will be easier to understand without having to separate .then.

```suggestion
const loaded = () => {
    const loadingPlace = document.getElementById("loadingPlace");
    loadingPlace.remove();
}

//Abbreviation

getData.then((value) => {
　   loaded();
    const ul = document.createElement('ul');
    const fragment = document.createDocumentFragment();

    value.forEach(item => {
        const li = document.createElement('li');
        const a = document.createElement('a');
        const img = document.createElement('img');

        a.textContent = item.text;
        a.href = item.to;
        img.src = item.img;
        img.alt = item.alt;

        fragment.appendChild(li).appendChild(a).appendChild(img);
    })
    div.appendChild(ul).appendChild(fragment);
});
```

Reply:
The code will be easier to read!
I want to deepen my understanding of grouping into functions.
I learned a lot!
Thank you very much!!